src/assets/images/Icons/MinusIcon.tsx

```diff
@@ -45,4 +45,4 @@ const MinusIcon = () => (
   </svg>
 )
 
-export default MinusIcon
+export default MinusIcon;
```

Comment:
Add Eslint rule for avoiding ; usage 
Do not use ;

Reply:
Done